discordbot/cogs/auditlogs.py

```diff
@@ -0,0 +1,101 @@
+import discord
+from discord.ext import commands
+
+intents = discord.Intents.default()
+intents.guilds = True
+intents.messages = True
+
+class AuditLog(commands.Cog):
+    def __init__(self, bot):
+        self.bot = bot
+    
+    async def is_admin_or_executive_check(ctx):
+        # Check if the user is an admin
+        if ctx.author.guild_permissions.administrator:
+            return True
+
+        # Check if the user has the 'executive' role
+        return any(role.name.lower() == 'executive' for role in ctx.author.roles)
+
+    @commands.command(name='printlogs', aliases=['PrintLogs'])
```

Comment:
for clarity's sake perhaps it would be best to add the word audit to the command so as not to confuse it

Reply:
renamed to auditlogs, call with: !auditlogs @user <num>